extensions/rocksdb-repos/RocksDbStream.cpp

```diff
@@ -28,19 +29,15 @@ namespace nifi {
 namespace minifi {
 namespace io {
 
-RocksDbStream::RocksDbStream(const std::string &path, rocksdb::DB *db, bool write_enable)
+RocksDbStream::RocksDbStream(std::string path, rocksdb::DB *db, bool write_enable)
     : BaseStream(),
-      path_(path),
+      path_(std::move(path)),
       write_enable_(write_enable),
       db_(db),
       logger_(logging::LoggerFactory<RocksDbStream>::getLogger()) {
   rocksdb::Status status;
   status = db_->Get(rocksdb::ReadOptions(), path_, &value_);
-  if (status.ok()) {
-    exists_ = true;
-  } else {
-    exists_ = false;
-  }
+  exists_ = status.ok();
```

Comment:
You don't even need "status" here, Get(...).ok() works as well.

Reply:
indeed, will remove